include/rppdefs.h

```diff
@@ -223,6 +223,14 @@ typedef struct
     Rpp32f data[6];
 } Rpp32f6;
 
+/*! \brief RPP 9 float vector
+ * \ingroup group_rppdefs
+ */
+typedef struct
+{
+    Rpp32f data[9];
+} Rpp32f9;
```

Comment:
Yes, for now as long as the external API takes just Rpp32f* and not Rpp32f9*, lets possibly keep it that way since warp_affine was also mostly written with Rpp32f6*
If need be, we'll change it for both